strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/HostedRepositoryProvider.java

```diff
@@ -55,24 +57,28 @@ public String getAlias()
     public ArtifactInputStream getInputStream(String storageId,
```

Comment:
I think that this method implementation should use `getPath` as we agreed in channel